build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation('com.github.rjeschke:txtmark:0.13')
     implementation('com.github.scopt:scopt_2.13:4.1.0')
     implementation('com.google.guava:guava:33.2.0-jre')
+    implementation 'org.apache.commons:commons-lang3:3.17.0'
```

Comment:
Is there a reason for the inconsistent syntax?

Reply:
Copy paste

Reply:
I had little time,  should be cleaned up